nupic/torch/modules/sparse_weights.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, module, weight_sparsity=None, sparsity=None):
                 DeprecationWarning,
             )
 
-        assert 0 < sparsity < 1
+        assert 0 <= sparsity <= 1
```

Comment:
Why do you need these edge cases? 

`sparsity=0` means dense
`sparsity=1` means all zeros

Not sure if any of these cases make sense in the context of the `SparseWeights`